tests/robustness/README.md

```diff
@@ -0,0 +1,71 @@
+# etcd Robustness Testing
+
+Purpose of etcd robustness tests is to validate that etcd upholds
+[API guarantees] and [watch guarantees] under any condition or failure.
+
+Robustness tests achieve that comparing etcd cluster behavior against a simplified model.
+Multiple test encompass different etcd cluster setups, client traffic types and failures experienced by cluster.
+During a single test we create a cluster and inject failures while sending and recording client traffic.
+Correctness is validated by running collected history of client operations against the etcd model and a set of validators.
+Upon failure tests generate a report that can be used to attribute whether failure was caused by bug in etcd or test framework. 
+
+[API guarantees]: https://etcd.io/docs/v3.6/learning/api_guarantees/
+[watch guarantees]: https://etcd.io/docs/v3.6/learning/api/#watch-streams
```

Comment:
cc @spzala to double confirm the link. Can we count on "`latest`"? 

```suggestion
[API guarantees]: https://etcd.io/docs/latest/learning/api_guarantees/
[watch guarantees]: https://etcd.io/docs/latest/learning/api/#watch-streams
```

Reply:
Done.